specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-10-01-preview/BulkActions.json

```diff
@@ -0,0 +1,652 @@
+              "value": "And"
+            },
+            {
+              "description": "Conditions/clauses should be joined together using logical OR operator.",
+              "value": "Or"
+            }
+          ]
+        }
+      },
+      "Clause": {
+        "description": "Represents a clauses in a condition.",
+        "type": "object",
+        "properties": {
+          "clauseConnective": {
+            "description": "Connective used to logically join the values on this clause",
+            "$ref": "#/definitions/Connective"
+          },
+          "field": {
+            "description": "The name of the field we are targeting.",
+            "type": "string"
+          },
+          "operator": {
+            "description": "Operator which describes how this clause should be constructed into an expression.",
+            "type": "string",
+            "enum": [
+              "Equal",
+              "NotEqual",
+              "LessThan",
+              "GreaterThan",
+              "StringContains",
+              "StringNotContains",
+              "IsNull",
+              "IsNotNull"
+            ],
+            "x-ms-enum": {
+              "modelAsString": true,
+              "name": "ConditionOperator",
+              "values": [
+                {
+                  "description": "The '=' operator.",
+                  "value": "Equal"
+                },
+                {
+                  "description": "The '!=' operator.",
+                  "value": "NotEqual"
+                },
+                {
+                  "description": "The '<' operator.",
+                  "value": "LessThan"
+                },
+                {
+                  "description": "The '>' operator.",
+                  "value": "GreaterThan"
+                }, 
+                {
+                  "description": "The 'contains' operator for strings.",
+                  "value": "StringContains"
+                },       
+                {
+                  "description": "The 'not contains' operator for strings.",
+                  "value": "StringNotContains"
+                },              
+                {
+                  "description": "The 'is_null' operator.",
+                  "value": "IsNull"
+                },
+                {
+                  "description": "The 'not is_null' operator.",
+                  "value": "IsNotNull"
+                }
+              ]
+            }
+          },
+          "values": {
+            "description": "Represents all values to construct this clause from in combination with the clause's field and operator.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "x-ms-identifiers": []
+          }
+        }
+      },
+      "Condition": {
+        "description": "Represents a condition which is composed of a set of clauses.",
```

Comment:
>Represents a condition which is composed of a set of clauses. [](http://example.com/codeflow?start=24&length=61)

If your service is going to reject duplicate clauses in this collection, pl mention it here.